src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+public class Ball {
+    String ballColor;
+    int ballNumber;
+
+    public Ball() {
+    }
+
+    public Ball(String color, int number) {
```

Comment:
No need to declare a construcor with arguments. There are multiple ways of declaring and intiating an object in java: by getter/setters, argumented constructor, builder pattern, etc. Please stick only to one approach. In this task you can go with the getters/setters approach but remebr to only expose needed operations.